app/src/main/java/com/duckduckgo/app/brokensite/api/BrokenSiteSender.kt

```diff
@@ -84,5 +87,6 @@ class BrokenSiteSubmitter(
         private const val MODEL_KEY = "model"
         private const val WEBVIEW_VERSION_KEY = "wvVersion"
         private const val SITE_TYPE_KEY = "siteType"
+        private const val GPC = "gpc"
```

Comment:
FYI doing this doesn't really do anything, we need to change the perl script that aggregates the data, otherwise that information will get lost.

Reply:
I've tried to move the code around like I said to `shouldInterceptRequest` and made some changes on a separate branch `feature/marcos/gpc_default_on` based on what I've been doing to change the user agent on a separate project. I think it works or at least is pretty close

@cmonfortep we'll catch up on Monday because I'm not 100% convinced if my version is breaking https://duckduckgo.github.io/privacy-test-pages/security/spoof-js-page-rewrite.html (I kinda think is not)

Reply:
Yes, we know that, we added it just to advance some work. See comments in https://app.asana.com/0/1198504124118156/1198503585911423/f

Reply:
Cool, thanks!